gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -2018,6 +2001,65 @@ gst_tensor_src_iio_fixate (GstBaseSrc * src, GstCaps * caps)
   return gst_caps_fixate (updated_caps);
 }
 
+/**
+ * @brief Perform state change.
+ */
+static GstStateChangeReturn
+gst_tensor_src_iio_change_state (GstElement * element,
+    GstStateChange transition)
+{
+  GstTensorSrcIIO *self;
+  GstStateChangeReturn ret;
+  gboolean buffer_state_change_success = TRUE;
+  gchar *dirname = NULL;
+
+  self = GST_TENSOR_SRC_IIO (element);
+
+  switch (transition) {
+    case GST_STATE_CHANGE_PAUSED_TO_PLAYING:
+    {
```

Comment:
You ain't need a new stack here. ( ```{```, ```}``` creates a stack although a "smart" compiler optimization techniques may remove such a stack)